src/Core/Cache/DefaultCacheFactory.php

```diff
@@ -2,17 +2,18 @@
 
 namespace SilverStripe\Core\Cache;
 
+use InvalidArgumentException;
 use Psr\Log\LoggerAwareInterface;
 use Psr\Log\LoggerInterface;
+use Psr\Cache\CacheItemPoolInterface;
 use Psr\SimpleCache\CacheInterface;
```

Comment:
```suggestion
use Psr\Cache\CacheItemPoolInterface as Psr6CacheInterface;
use Psr\SimpleCache\CacheInterface as Psr16CacheInterface;
```
The actual interface names aren't very useful, since all we really care about is which standard each of these represents. Let's make is really really clear which interface is for which standard.

Reply:
I can see why we'd do this, though I'm also not really a fan of this

While it makes things clearer, I think it also makes things more confusing as we're introducing new names.

For instance we also have keys defined in YML such as: `Psr\SimpleCache\CacheInterface.Sha1FileHashingService` - I'd rather consistently just call all of the PSR16 things "CacheInterface", even though there's a collision with Symfony's definition of a `CacheInterface`

CacheInterface and CacheItemPoolInterface ARE the standard names in the php ecosystem so I'd prefer to just use them.

Reply:
That's fair. This is a subjective piece of syntax sugar and the path of least resistance is to leave as is so I'm okay with that.